packages/cli/lib/services/dryrun.service.ts

```diff
@@ -220,6 +244,10 @@ class DryRunService {
                 }
```

Comment:
Since you are buffering the output, the prompt will pop before any messages has been printed I think

Reply:
Not sure I follow -- I'm seeing this work as expected.

Reply:
yes sorry I thought you removed the console.log 🤦🏻